.cicd/pipeline.yml

```diff
@@ -3,78 +3,100 @@ env:
   TEST_TIMEOUT: 60
   TIMEOUT: 120
   VERBOSE: true
+  DEBUG: true
 
 steps:
   - wait
 
   - label: ":webhook: Trigger Travis CI Build"
-    command: |
-      . ./.cicd/trigger-travis.sh
+    command:
+      - "./.cicd/travis/travis-trigger.sh"
     agents:
-      queue: "automation-eos-builder-fleet"
+      queue: "automation-basic-builder-fleet"
 
   - trigger: "eosio-base-images-beta"
-    label: ":docker: Ensure base images exist"
+    label: ":docker: Ensure Base Images"
     build:
       commit: "${BUILDKITE_COMMIT}"
       branch: "${BUILDKITE_BRANCH}"
 
   - wait
 
-  - label: ":aws: Amazon_Linux 2 - Build"
+  - label: ":aws: Amazon_Linux 2 - Build & Test"
     agents:
       queue: "automation-eos-builder-fleet"
     plugins:
       - docker#v3.2.0:
           debug: $DEBUG
-          image: "eosio/producer:eosio-amazonlinux-2-1c266e25276ad1f1147a4df5c12921968079c49b"
+          image: "eosio/producer:eosio-amazonlinux-2-cc8d19ea479663d98189ce2473aacc2a7567a8be"
+          environment:
+            - "BUILDKITE_AGENT_ACCESS_TOKEN" # Needed for buildkite-agent upload/download
+          mount-buildkite-agent: false # Mounting bk-agent doesn't work, so disable it
+          propagate-environment: true # Need for buildkite-agent upload/download (JOB_ID, etc)
     timeout: $BUILD_TIMEOUT
     skip: $SKIP_AMAZON_LINUX_2
 
-  - label: ":centos: CentOS 7.6 - Build"
+  - label: ":centos: CentOS 7 - Build & Test"
     command:
       - ""
     agents:
       queue: "automation-eos-builder-fleet"
     plugins:
       - docker#v3.2.0:
-          debug: $DEBUG
-          image: "eosio/producer:eosio-centos-7-d45789dfb1a5b830a62381afd205b34d3d7c2d63"
+          debug: true
+          image: "eosio/producer:eosio-centos-7-61150e3338126b91bd9384821be93de5e9f8b3fe"
+          environment:
+            # - "ENABLE_LR_TESTS=true"
```

Comment:
Remove commented-out code.

Reply:
It's there to show how it works/demo. I can remove it later

Reply:
Remove commented-out code before merge.
This is the point of version control, [that code](https://github.com/EOSIO/eos/blob/3b21fdd536aa9057f98d4bfaac594dfa079735fd/.cicd/pipeline.yml#L42) is never gone.